sympy/solvers/ode/ode.py

```diff
@@ -346,35 +347,6 @@
     )
 
 
-def sub_func_doit(eq, func, new):
-    r"""
-    When replacing the func with something else, we usually want the
-    derivative evaluated, so this function helps in making that happen.
-
-    Examples
-    ========
-
-    >>> from sympy import Derivative, symbols, Function
-    >>> from sympy.solvers.ode.ode import sub_func_doit
-    >>> x, z = symbols('x, z')
-    >>> y = Function('y')
-
-    >>> sub_func_doit(3*Derivative(y(x), x) - 1, y(x), x)
-    2
-
-    >>> sub_func_doit(x*Derivative(y(x), x) - y(x)**2 + y(x), y(x),
-    ... 1/(x*(z + 1/x)))
-    x*(-1/(x**2*(z + 1/x)) + 1/(x**3*(z + 1/x)**2)) + 1/(x*(z + 1/x))
-    ...- 1/(x**2*(z + 1/x)**2)
-    """
-    reps= {func: new}
-    for d in eq.atoms(Derivative):
-        if d.expr == func:
-            reps[d] = new.diff(*d.variable_count)
-        else:
-            reps[d] = d.xreplace({func: new}).doit(deep=False)
-    return eq.xreplace(reps)
-
 
```

Comment:
How many blank lines are left? Does it match the spacing between functions in other parts of the module?